_layouts/default.html

```diff
@@ -1,4 +1,5 @@
 {% include head.html %}
 {% include header.html %}
+<script defer src="https://use.fontawesome.com/releases/v5.1.0/js/all.js" integrity="sha384-3LK/3kTpDE/Pkp8gTNp2gR/2gOiwQ6QaO7Td0zV76UFJVhqLl4Vl3KL1We6q6wR9" crossorigin="anonymous"></script>
```

Comment:
can you put this in the head.html include

Reply:
Thanks, let me ammend it :)